test/unit/presubmit-tests.sh

```diff
@@ -42,6 +42,10 @@ function init_test_env() {
   function check_links_in_markdown() {
     return 0
   }
+
+  function default_build_test_runner() {
+    return 0
```

Comment:
You're not checking that the default runner was executed...

Reply:
Added another test for it